core/src/Platforms/Android/AndroidPlatformProxy.cs

```diff
@@ -134,6 +134,25 @@ public string GetProductName()
         {
             return _isMsal ? "MSAL.Xamarin.Android" : "PCL.Android";
         }
+        public string GetCallingApplicationName()
+        {
+            return Android.App.Application.Context.ApplicationInfo?.LoadLabel(Android.App.Application.Context.PackageManager);
+          
```

Comment:
nit: remove extra blank line here and 140 and 146 and 154

Reply:
done